force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -31,10 +31,19 @@ public with sharing class VariablesDataTypesOperators {
      * @param b The second number.
      * @return The sum of the two numbers, or null if either number is null.
      */
-    public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
-    }
-
+    public static Integer addition(Integer a, Integer b)
+    
+    { 
+         Integer result;          
+           if (a == null || b == null)
+           {
+                return null;
+            }
+                return result =  a + b;
+                   }
+    
```

Comment:
Try to see if you can line up these parenthesis, so they are easy to read. I use a vs code extension call indent-rainbow to do this.